domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/service/ReviewService.kt

```diff
@@ -128,25 +128,26 @@ class ReviewService(
 
     // Calculate the review schedule window and rule
     val releaseDate = createInitialReviewScheduleDto.prisonerReleaseDate
+    val sentenceType = createInitialReviewScheduleDto.prisonerSentenceType
     val reviewScheduleCalculationRule = determineReviewScheduleCalculationRule(
-      sentenceType = createInitialReviewScheduleDto.prisonerSentenceType,
+      sentenceType = sentenceType,
       releaseDate = releaseDate,
       isReAdmission = createInitialReviewScheduleDto.isReadmission,
       isTransfer = createInitialReviewScheduleDto.isTransfer,
     )
-    val reviewScheduleWindow = calculateReviewWindow(reviewScheduleCalculationRule, releaseDate)
-      ?: throw RuntimeException("Unable to determine review schedule for release date $releaseDate.")
```

Comment:
It's not an exception if the review schedule window could not be calculated; it just means that its not necessary/appropriate to create a review schedule for this prisoner